com.rustdesk.RustDesk.json

```diff
@@ -0,0 +1,110 @@
+{
+  "id": "com.rustdesk.RustDesk",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "rustdesk",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=network",
+    "--filesystem=home",
+    "--device=dri",
+    "--socket=pulseaudio"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/share/gtk-doc"
+  ],
+  "modules": [
+    "shared-modules/libappindicator/libappindicator-gtk3-12.10.json",
+    {
+      "name": "xdotool",
+      "buildsystem": "simple",
+      "build-commands": [
+        "make -j4 && PREFIX=./build make install",
+        "cp -r ./build/*  /app/"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz",
+          "sha256": "96f0facfde6d78eacad35b91b0f46fecd0b35e474c03e00e30da3fdd345f9ada"
+        }
+      ]
+    },
+    {
+      "name": "pam",    
+      "buildsystem": "simple",
+      "build-commands": [
+          "./configure --disable-selinux --prefix=/app && make -j4 install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/linux-pam/linux-pam/releases/download/v1.3.1/Linux-PAM-1.3.1.tar.xz",
+          "sha256": "eff47a4ecd833fbf18de9686632a70ee8d0794b79aecb217ebd0ce11db4cd0db"
+        }
+      ]
+    },
+    {
+      "name": "rustdesk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "bsdtar -zxvf rustdesk.deb",
+        "tar -xvf ./data.tar.xz",
+        "cp -r usr/* /app/",
+        "mkdir -p /app/bin && ln -s /app/lib/rustdesk/rustdesk /app/bin/rustdesk",
+        "mv /app/share/applications/rustdesk.desktop /app/share/applications/com.rustdesk.RustDesk.desktop",
+        "sed -i '/^Icon=/ c\\Icon=com.rustdesk.RustDesk' /app/share/applications/*.desktop",
+        "mv /app/share/icons/hicolor/scalable/apps/rustdesk.svg /app/share/icons/hicolor/scalable/apps/com.rustdesk.RustDesk.svg",
```

Comment:
leave the icon alone, remove the sed line (don't use sed anyway), leave the desktop file alone and then use `rename-desktop-file` and `rename-icon` in the manifest to take care of the renaming.

Reply:
@hfiguiere are you sure about this? I see there is an issue with these properties: https://github.com/flathub/org.flatpak.Builder/issues/71

I am trying to do it too on Ubuntu 24.04 and my `.desktop` and `.svg` file is not renaming.
flatpak-builder 1.4.4

Reply:
1/ use the flatpak.
2/ pretty sure, yes.

Reply:
can you fix this too ? Just do `rename-desktop-file: rustdesk.desktop` and `rename-icon: rustdesk` on top of the manifest

Reply:
done.

@bbhtt thank you! Your comment is really helped me.